### ISSUE TITLE
OZW patch version number (1.6.####) missing from Docker build

### ISSUE THREAD

**Issue by visnaut** [robertsLando/Zwave2Mqtt-docker#12]
This not may not be an issue, but I'd like to verify: when I run the docker container with either the `latest` or `2.1.1` tags, the "Openzwave" field shows a version number of 1.6.0. However, in the README, it's [currently specified](https://github.com/robertsLando/Zwave2Mqtt-docker/commit/e028abe6d3a795679936f10ee8ab326b84eed044#diff-04c6e90faac2675aa89e2176d2eec7d8R33) that the version should be 1.6.1004.

At first I figured this may just be a UI issue, but when I run the Z2M package locally, the field does in fact show my locally-installed OZW version: 1.6.962

Just looking to verify whether the version running in the docker container is correct before I start triaging issues I'm seeing with my devices. Thanks in advance!

**Comment by visnaut** [comment]
Did some further digging: If you check https://hub.docker.com/r/robertslando/zwave2mqtt/builds, it looks like the last successful build of this container was 9 months ago. 

Not sure if this is intentional, but given that the README was updated so recently, I'm assuming not?

**Comment by visnaut** [comment]
I decided to do a little bit more digging and built the image myself. Here's what I've found:

The patch version number in the docker container is missing because Open Z-Wave's build scripts relies on git, and the checked out `OpenZWave/open-zwave` repository, [to determine the `VERSION_REV`](https://github.com/OpenZWave/open-zwave/blob/9526554bdffd9e2049b3e9ad1ab0d119fdb9595f/cpp/build/support.mk#L45). Since the current Dockerfile doesn't install `git` and [downloads the tar archive directly](https://github.com/robertsLando/Zwave2Mqtt-docker/blob/master/Dockerfile#L34), then that value defaults to zero. I've updated this ticket's title to more accurately reflect this.

I'm considering throwing a PR your way that would instead install git, and do a shallow clone at the desired commit SHA. Let me know if you'd be willing to merge that in, @robertsLando.

As a side note, when I modified my build to these SHAs, I resolved all of the device issues I was having:
```sh
Z2M_GIT_SHA1=5bbb7f6542576d4e3c45045241b5b904eb1560f5
OPENZWAVE_GIT_SHA1=0d94c9427bbd19e47457578bccc60b16c6679b49
```
